### PR TITLE
Fixed #29864 -- Added link targets for low-level cache API.

### PR DESCRIPTION
**Pull request by nixphix**
https://code.djangoproject.com/ticket/29864

### CODE REVIEW

**Comment by timgraham** [comment]
My patch wasn't meant to be complete.

File: docs/topics/cache.txt [hunk 1]
```diff
@@ -820,7 +827,9 @@ return if the object doesn't exist in the cache::
 
 To add a key only if it doesn't already exist, use the ``add()`` method.
 It takes the same parameters as ``set()``, but it will not attempt to
-update the cache if the key specified is already present::
+update the cache if the key specified is already present:
+
+.. method:: cache.add(key, value, timeout)
```
Comment (timgraham) [inline docs/topics/cache.txt]: These lines should go above the paragraph that describe its behavior.

Comment (nixphix) [reply docs/topics/cache.txt]: Done, should I follow the same for rest of the methods as well ?

Comment (timgraham) [reply docs/topics/cache.txt]: Yes please.

Comment (nixphix) [reply docs/topics/cache.txt]: done

**Comment by timgraham** [comment]
The text and code shouldn't be reordered. The order should be link anchor, text, code example.

**Comment by nixphix** [comment]
> The text and code shouldn't be reordered. The order should be link anchor, text, code example.

Done, except for incr/decr methods because they have common text.

**Comment by timgraham** [comment]
You can put the link anchors for incr and decr next to each other, followed by the text and code examples. Please revert the punctuation changes that are now in the patch.

**Pull request merged**